sql/1.0xx/activity_feed.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+CREATE TABLE "activity_feed" (
+  "id" serial NOT NULL,
+  "created" timestamp with time zone NOT NULL,
+  "type" text NOT NULL,
+  "description" text NOT NULL,
+  "format" text DEFAULT 'markdown' NOT NULL,
+  "for_user" bigint,
```

Comment:
The idea of adding the same notification for a handful of users is OK, I think (e.g. mailing stats to 3 or 4 people).

The idea of `for_user` is to target stuff like replies,@ -mentions....